rustfmt-core/rustfmt-lib/src/closures.rs

```diff
@@ -25,7 +25,7 @@ use crate::utils::{last_line_width, left_most_sub_expr, stmt_expr, NodeIdExt};
 
 pub(crate) fn rewrite_closure(
     capture: ast::CaptureBy,
-    is_async: &ast::IsAsync,
+    is_async: &hir::IsAsync,
```

Comment:
Do we need `hir`? Figured this would still be the [Async enum]( https://docs.rs/rustc-ap-rustc_ast/649.0.0/rustc_ap_rustc_ast/ast/enum.Async.html) from ast